src/index.js

```diff
@@ -340,7 +340,20 @@ class MiniCssExtractPlugin {
                   'var linkTag = document.createElement("link");',
                   'linkTag.rel = "stylesheet";',
                   'linkTag.type = "text/css";',
-                  'linkTag.onload = resolve;',
+                  'linkTag.href = fullhref;',
+                  "// old webkit's would claim to have onload, but didn't really support it",
+                  '// https://github.com/kriszyp/xstyle/blob/master/core/load-css.js',
+                  'var webkitVersion = navigator.userAgent.match(/AppleWebKit\\/(\\d+\\.?\\d*)/);',
```

Comment:
This extra check affects only 0.02% of the internet. I don't think it's worth including this workaround of everybody.